### ISSUE TITLE
Investigate using Sox

### ISSUE THREAD

**Issue by Molkree** [orcasound/orca-action-workflow#20]
@wetdog suggested using [Sox](http://sox.sourceforge.net/sox.html) to generate [spectrograms](http://sox.sourceforge.net/sox.html#:~:text=plot%20global%20option.-,spectrogram,-%5Boptions%5D).
What are (dis)advantages over `obspy`/`scipy`? If there is no processing we could just drop Python and use bash+Sox I guess (or even some light processing, see link above).

float32 spectrogram:
![live000_float32_spectrogram](https://user-images.githubusercontent.com/12860284/122121444-5420f780-ce34-11eb-9194-33a864ccd420.png)

**Comment by Molkree** [comment]
So turns out that Sox is actually pretty [fast](https://github.com/Molkree/orca-action-workflow/actions/runs/955527641)

![image](https://user-images.githubusercontent.com/12860284/122696141-0a2c7d00-d24b-11eb-8e92-980b312a5963.png)

**Comment by wetdog** [comment]
That's great, I think that you can get rid of the colorbar and the axes labels with the flag -r

**Comment by Molkree** [comment]
Yup, that's true, `-r` will create a raw spectrogram. But @valentina-s  actually requested to add dB scale to the spectrograms generated using Python (#6) so we'll probably keep them. And as @yosoyjay correctly noted if we need raw spectrograms for later processing it's really easy to cut them out from the image.

But for now, we are sticking with Python because it is easier to use different tools operating on the raw data, rather than using Sox + something else on images.
